### PR TITLE
Add log appender / bridge to glossary

### PR DESCRIPTION
**Pull request by jack-berg**
Related to #2911.

cc @dyladan who brought up naming in the 3/13 maintainers SIG.

### CODE REVIEW

**Review by tigrannajaryan — APPROVED** [review]

**Comment by dyladan** [comment]
I thought in the meeting we agreed to use only one term. Using both terms is unlikely to reduce user confusion.

**Comment by jack-berg** [comment]
As much as I'd like to have a single term, after thinking about it more, I think there's too much momentum behind both terms. Appender is a popular term in the logging domain and is used in multiple language ecosystems. Authors of these components are going to inevitably call them appenders. Bridge is a useful term in the spec, and is used in the opencensus, opentracing, and metric producer specs. These components are definitely bridges in the same sense the term is used elsewhere. 

I think we should standardize on the term "bridge" across the spec to refer to any component that bridges data into OpenTelemetry from another API / framework, but also be ok with aliases for these components that reflect the domain.

**Review by cijothomas — APPROVED** [review]

**Comment by reyang** [comment]
> I think we should standardize on the term "bridge" across the spec to refer to any component that bridges data into OpenTelemetry from another API / framework, but also be ok with aliases for these components that reflect the domain.

Personally (very subjective), I prefer "provider" than "bridge" because "bridge" sounds like a bi-directional thing to me. But I'm 100% supportive of whatever term as long as we 1) make it explicit/clear 2) stick to the one that we picked.

**Review by carlosalberto — APPROVED** [review]

File: specification/logs/bridge-api.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 <b>Note: this document defines a log *backend* API. The API is not intended to be called
 by application developers directly. It is provided for logging library authors
-to build [Appenders](#how-to-create-log4j-style-appender), which use
+to build [log appenders](#how-to-create-log4j-log-appender), which use
```
Comment (arminru) [inline specification/logs/bridge-api.md:30]:
```suggestion
to build [log appenders](#how-to-create-a-log4j-log-appender), which use
```

File: specification/logs/bridge-api.md [hunk 2]
```diff
@@ -164,15 +164,16 @@ fields:
 
 ## Usage
 
-### How to Create Log4J Style Appender
+### How to Create Log4J Log Appender
```
Comment (arminru) [inline specification/logs/bridge-api.md:167]:
```suggestion
### How to Create a Log4J Log Appender
```

**Review by arminru — APPROVED** [review]

**Pull request merged**